OpenRA.Mods.D2k/Traits/AutoCarryall.cs

```diff
@@ -30,9 +34,12 @@ public class AutoCarryall : INotifyBecomingIdle, INotifyKilled, ISync, IRender
 	{
 		readonly Actor self;
 		readonly WRange carryHeight;
+		readonly AutoCarryallInfo info;
 
 		// The actor we are currently carrying.
-		[Sync] Actor carrying;
+		[Sync] public Actor Carrying { get; internal set; }
+
+		public bool HasCarryableAttached { get; internal set; }
```

Comment:
Actually I prefer `HasCarryableAttacked` over `Carrying`. The latter doesn't say much and doesn't sound as good. I don't think a property name that explains *too well* what it's for should be a problem.